doc/releases/changelog-dev.md

```diff
@@ -183,21 +213,12 @@
   for certain special cases of Pauli strings (e.g. for `XX` and `YY` type Pauli strings).
   [(#8562)](https://github.com/PennyLaneAI/pennylane/pull/8562)
 
-* Added a new decomposition, `_decompose_2_cnots`, for the two-qubit decomposition for `QubitUnitary`.
-  It supports the analytical decomposition a two-qubit unitary known to require exactly 2 CNOTs.
-  [(#8666)](https://github.com/PennyLaneAI/pennylane/issues/8666)
-
-* Quantum compilation passes in MLIR and XDSL can now be applied using the core PennyLane transform
-  infrastructure, instead of using Catalyst-specific tools. This is made possible by a new argument in
-  :func:`~pennylane.transform` and `~.TransformDispatcher` called ``pass_name``, which accepts a string
-  corresponding to the name of the compilation pass.
-  The ``pass_name`` argument ensures that the given compilation pass will be used when qjit'ing a
-  workflow, where the pass is performed in MLIR or xDSL.
-  [(#8539)](https://github.com/PennyLaneAI/pennylane/pull/8539)
+* Users can now estimate the resources for quantum circuits that contain or decompose into
+  any of the following symbolic operators: :class:`~.ChangeOpBasis`, :class:`~.Prod`,
+  :class:`~.Controlled`, :class:`~.ControlledOp`, :class:`~.Pow`, and :class:`~.Adjoint`.
+  [(#8464)](https://github.com/PennyLaneAI/pennylane/pull/8464)
 
-* `Operator.decomposition` will fallback to the first entry in `qml.list_decomps` if the `Operator.compute_decomposition`
-  method is not overridden.
-  [(#8686)](https://github.com/PennyLaneAI/pennylane/pull/8686)
+<h4>Inspecting your compilation pipeline</h4>
```

Comment:
I think this feature along with specs (coming in from here: https://github.com/PennyLaneAI/pennylane/pull/8606) and graph visualization (no PR yet but it's coming) can have their own section within the New features major section 👀 

Could be something like "Seamless resource tracking and circuit visualization for compiled programs" or something?